juriscraper/opinions/united_states/state/coloctapp.py

```diff
@@ -34,12 +34,14 @@ def _process_html(self) -> None:
         date = rows[0].text_content()
         date_year = convert_date_string(date)
 
-        if "P U B L I S H E D  O P I N I O N S" != rows[1].text_content():
+        if "P U B L I S H E D O P I N I O N S" != clean_string(
+            rows[1].text_content()
+        ):
             return {}
```

Comment:
Does line 40 make sense.  I introduced it, but I dont understand it now.

Reply:
It stops the scraper in case there aren't published opinions to read in the current week.
I've seen that in some past weeks, the Colorado Appellate only releases unpublished opinions. We don't have a way to get the URL for those opinions' files, so, they are ignored.